### DISCUSSION TITLE
How to set up geometry for pupil foreshortening error correction for combined eyetracking / fMRI experiment

### DISCUSSION THREAD

**Discussion by amjh23** [bachlab/PsPM#557] [Q&A] [answered]
Dear experts,
I am unsure how to define the geometry for doing pupil foreshortening error (PFE) corection of my pupil area data collected inside an MR scanner: We actually tracked the right eye, but because the camera saw the eye through a mirror, for the camera it looked like we were tracking a left eye. Is it correct in this case to tell pspm that we tracked the left eye, and to report all positions of camera, monitor and eye the way the camera saw them, not the way the eye saw them? 
Many thanks for your help!

**Comment by dominikbach** [comment, marked as answer]
Dear all
PsPM implements the foreshortening error correction published in 
Hayes, T. R & Petrov, A. A. (2015) Mapping and correcting the influ- ence of gaze position on pupil size measurements. Behav. Res. Methods 48, 510–527.
If in doubt, we would recommend getting in touch with the authors of the method.
The PsPM team so far has not used or tested this method with mirror systems.
Dominik
